app/modules/sightings/parameters.py

```diff
@@ -46,14 +49,19 @@ class PatchSightingDetailsParameters(PatchJSONParameters):
         '/verbatimLocality',
     )
 
-    PATH_CHOICES_HOUSTON = ('/assetId', '/newAssetGroup')
+    PATH_CHOICES_HOUSTON = ('/assetId', '/newAssetGroup', '/featured_asset_guid')
```

Comment:
everything else is caMeLCaSe, this isn't, seems oddly inconsistent

Reply:
I think everything else is in the EDM- which is java and likes camelCase. This particular field is in python Houston where snake case is preferred. I see the argument for consistency and have changed it though.